### PR TITLE
Wish for Kirontoo

### PR DESCRIPTION
**Pull request by kirontoo**

### CODE REVIEW

File: wishing-wall/kirontoo-wish [hunk 1]
```diff
@@ -0,0 +1 @@
+this is my wish
```
Comment (monarchwadia) [inline wishing-wall/kirontoo-wish:1]: Can you put a real wish? :-)

**Review by monarchwadia — CHANGES_REQUESTED** [review]

**Comment by kirontoo** [comment]
just changed my wish!

**Review by monarchwadia — APPROVED** [review]

**Review by monarchwadia — APPROVED** [review]
Your wish has been granted :-P

**Comment by monarchwadia** [comment]
Congrats on the merge!

**Pull request merged**